src/custom-elements/src/utils/Drag.ts

```diff
@@ -116,17 +124,19 @@ export class Drag {
   }
 
   private handlerResize = () => {
-    const rect = this.elementToMove.getBoundingClientRect();
-
-    this.screenWidth = this.setScreenWidth();
-    this.screenHeight = this.setScreenHeight();
-
-    if (rect.right >= this.screenWidth) {
-      this.elementToMove.style.left =
-        Math.abs(rect.width + rect.width / 2 - this.screenWidth) + 'px';
-    } else if (rect.bottom >= this.screenHeight) {
-      this.elementToMove.style.top =
-        Math.abs(rect.height * 2 - this.screenHeight) + 'px';
+    if (this.elementToMove) {
```

Comment:
This way you don't add complexity to the block
```suggestion
    if (!this.elementToMove) {
    return;
    }
```